docs/webhooks/events.md

```diff
@@ -254,21 +254,21 @@ Type | Description
 
 Property | Value | Description
 ---------|-------|------------
-`answers` | Hash | Survey question and responses
 `submitted_at` | date-time | Date/time response was submitted (EST/EDT or GMT -5/GMT -4)
-`response_id` | String | Survey external Response ID
-`url` | Hash | URL variable value for Survey
 `main_indicator` | Integer | Main question result
 `main_indicator_scaled` | Float | Main question result scaled to 0..1
 `intervention_id` | String | ID of the intervention related to this survey
 `survey_id` | String | ID of the survey
 `source_id` | String | ID of the source related to this survey
 `user_id` | String | ID of the user related to this survey
-`context_data` | Hash | Context_data is present only if the content has context_data associated. The context_data hash keys are the custom fields keys.
+`questions` | Array of Hashes | List of questions and their responses. Each entry contains two keys: `text` and `value`
```

Comment:
> Each entry contains two keys: `text` and `value`

this part is not exactly right, here's the structure of the `questions` field:
```json
[
  {
    "foreign_id": "free_text_question",
    "text": "What can we do better next time?",
    "replies": [
      {
        "text": "Answer faster",
        "value": "Answer faster"
      }
    ]
  },
  {
    "foreign_id": "multiple_choice_question",
    "text": "What do you like to eat?",
    "replies": [
      {
        "text": "Fish",
        "value": "3"
      },
      {
        "text": "Eggs",
        "value": "4"
      }
    ]
  },
  {
    "foreign_id": "single_choice_question",
    "text": "How would you rate our response?",
    "replies": [
      {
        "text": "Good",
        "value": "5"
      }
    ]
  }
]
```

IMO we should specify that each element of the array represents a question and inside this element there's another array of hashes that represents each of the replies to the given question

Reply:
Oh yeah it's not very clear, let me fix that

Reply:
Did it in ebb77e9), I also updated the screenshot above of the description of the PR if you want to see the final result.